acceptance/src/main/java/io/github/tcdl/msb/acceptance/MsbTestHelper.java

```diff
@@ -71,47 +71,47 @@ public ObjectMapper getPayloadMapper() {
         return getPayloadMapper(DEFAULT_CONTEXT_NAME);
     }
 
-    public Requester<Payload<Object, Object, Object, Map<String, Object>>> createRequester(String namespace, Integer numberOfResponses) {
+    public Requester<RestPayload<Object, Object, Object, Map<String, Object>>> createRequester(String namespace, Integer numberOfResponses) {
```

Comment:
This class looks a bit more scary but it's going to be cleaned up during next task related to the update of tests. For most tests we need to have only a very simple string payload and hence don't need to bother with `RestPayload`